### PR TITLE
Baseline

### PR DESCRIPTION
**Pull request by karimkawambwa**
This serves to cleanup and refactor the map.

- [x] Determine if has url is required and resolve hash state update to draw hexLayer in the correct geo positions.

cc @kilemensi @nerdlet

### CODE REVIEW

**Comment by karimkawambwa** [comment]
@kilemensi  @nerdlet do you think they want the hash endpoint functionality ? We can just opt not to support the hash and remove Leaflet Hash. This way we can deal with just query params and not worry about the state. 

The only downside I see and argument to keep this is when someone wants to copy the url and share with someone else. The query params don't get updated but the hash does get updated by Leaflet Hash.

**Comment by kilemensi** [comment]
> @kilemensi @nerdlet do you think they want the hash endpoint functionality ? We can just opt not to support the hash and remove Leaflet Hash. This way we can deal with just query params and not worry about the state.
> 
> The only downside I see and argument to keep this is when someone wants to copy the url and share with someone else. The query params don't get updated but the hash does get updated by Leaflet Hash.

Ability to just copy and share URLs feels like a big deal to me @karimkawambwa.

 - Can you may be explain why Leaflet Hash isn't working the way it's supposed to & may be why _we_ can't fix it? and
 - If we're going the query params route, how hard would it be to support auto-updating of the query params to maintain the ability to just copy and share URLs.

**Comment by karimkawambwa** [comment]
> Ability to just copy and share URLs feels like a big deal to me @karimkawambwa.
> 
> * Can you may be explain why Leaflet Hash isn't working the way it's supposed to & may be why _we_ can't fix it? and
> * If we're going the query params route, how hard would it be to support auto-updating of the query params to maintain the ability to just copy and share URLs.

@kilemensi I am still looking int leaflet map and hex layer so I can resolve this. Its not straight forward where to fix it but the problem is when the hash changes, leaflet hash seems to move the map to the correct position however, the hex layer does not get translated to the correct position. The hex layer is just an svg layer. It get update when you move the map etc but when you change the hash it does not get translated accordingly. 

> If we're going the query params route, how hard would it be to support auto-updating of the query params to maintain the ability to just copy and share URLs.

I am not sure how easy it would be to update the query params without reloading the site, I am not sure that if this is possible, I have not tried this before.

**Comment by karimkawambwa** [comment]
@kilemensi @nerdlet 

I fix, please review, thanks 😃

**Comment by karimkawambwa** [comment]
@kilemensi @nerdlet ping

File: src/feinstaub-api.js [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
-
-import _ from "lodash";
-import "whatwg-fetch";
+import _ from 'lodash'
+import 'whatwg-fetch'
 
 let api = {
-  fetchNow() {
-    return fetch(URL).then(response => response.json());
+  fetchNow () {
```
Comment (kilemensi) [inline src/feinstaub-api.js]: Not sure about this space between `function` and `()`. Aren't we using [AirBnB style](https://github.com/airbnb/javascript#functions--signature-spacing)?

File: src/feinstaub-api.js [hunk 2]
```diff
@@ -1,40 +1,39 @@
 // const URL = 'https://api.luftdaten.info/static/v2/data.dust.min.json'
 // const URL = 'https://api.luftdaten.info/v1/filter/area=48.800000,9.200000,50'
 // const URL = 'https://api.airquality.codeforafrica.org/v2/sensors/?type=PPD42NS,HPM,PMS1003,PMS3003,PMS5003,PMS6003,PMS7003,SDS021,SDS011'
-const URL = "https://api.sensors.africa/v2/nodes/";
+const URL = 'http://api.sensors.africa/v2/nodes/?format=json'
+// const URL = "http://staging.api.sensors.africa/v2/nodes/";
```
Comment (kilemensi) [inline src/feinstaub-api.js]: Didn't I see somewhere you mentioning about having this in `.env` or something?

**Review by kilemensi — APPROVED** [review]
LGTM @karimkawambwa. Lets merge #17 and then update and merge this.

**Pull request merged**